dashboard/test/lib/pd/payment/payment_calculator_base_test.rb

```diff
@@ -147,7 +147,7 @@ class PaymentCalculatorBaseTest < ActiveSupport::TestCase
     end
 
     test 'teacher summaries' do
-      workshop = create :pd_ended_workshop, num_sessions: 2
+      workshop = create :pd_ended_workshop, :local_summer_workshop, num_sessions: 2
```

Comment:
This test was broken by the new time constraints (specifically, max days) on CSF workshops. Since the test doesn't seem to be testing for a CSF workshop in particular, I've updated it to run against a CSP local summer workshop instead.